db/migrations/81_rename_groups_attempts_to_attempts.sql

```diff
@@ -119,6 +78,8 @@ CREATE TRIGGER `before_delete_items_items` BEFORE DELETE ON `items_items` FOR EA
     FROM `permissions_generated`
     WHERE `permissions_generated`.`item_id` = OLD.`parent_item_id`;
 
+    DELETE FROM `attempts` WHERE `item_id` = OLD.`parent_item_id` AND `started_at` IS NULL AND `score_edit_rule` IS NULL;
+
     UPDATE `attempts` SET `result_propagation_state` = 'to_be_recomputed'
     WHERE `item_id` = OLD.`parent_item_id`;
```

Comment:
I think that is still not correct, if the parent has other children, and has null `started_at` and `score_edit_rule`, it will be deleted and so its score will not be recomputed. 

As you had done before, ancestors may have to be deleted as well somehow... but it is maybe complicated to compute which ones can be removed for sure, it would be the ones with no started_at, no score_edit_rule, and without children after applying the same criteria on children... maybe that's safer not to delete anything actually. 😕

Reply:
> I think that is still not correct, if the parent has other children, and has null started_at and score_edit_rule, it will be deleted and so its score will not be recomputed.